config/environments/production.rb

```diff
@@ -168,28 +168,11 @@
     end
   end
 
-  # Add throttling to application
-  require_relative '../../lib/rack_throttle/rules'
-
-  rules = [
-    { method: 'POST', limit: ENV['GRAPH_API_RATE_LIMIT'] },
-    { method: 'GET', whitelisted: true }
-  ]
-
-  cache =
-    if ENV['MEMCACHEDCLOUD_SERVERS'].present?
-      Dalli::Client.new(
-        (ENV['MEMCACHEDCLOUD_SERVERS']).split(','),
-        {
-          username: ENV['MEMCACHEDCLOUD_USERNAME'],
-          password: ENV['MEMCACHEDCLOUD_PASSWORD']
-        }
-      )
-    end
+  Rack::Attack.enabled = ENV['REDIS_CACHE_URL'].present?
 
-  config.middleware.use RackThrottle::Rules,
-                        rules: rules,
-                        cache: cache,
-                        key_prefix: :throttle,
-                        default: 20
+  Rack::Attack.cache.store =
+    ActiveSupport::Cache::RedisCacheStore.new(
+      namespace: 'throttle',
```

Comment:
Since we'll be sharing the Redis cluster between many apps, this namespace string should probably reference both the app and the purpose. So... `lms-throttle`?